contrib/tiflash-proxy-cmake/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+option(ENABLE_UNI_PS_FFI "Enable write proxy data to uni ps" OFF)
```

Comment:
Is the behavior "write proxy data to uni ps" controlled by this compile config but not a runtime config?

Reply:
Yes, but change it to a runtime config requires non-trivial change. We may do it in another pr.